scripts/release/helper/release_deb.sh

```diff
@@ -40,7 +39,7 @@ cat <<EOF>${HOME}/.aptly.conf
   "S3PublishEndpoints": {
     "algorand-releases": {
       "region":"us-east-1",
-      "bucket":"algorand-releases",
+      "bucket":"ben-test-release-bucket",
```

Comment:
we want to release it into the "algorand-build" bucket, and extend the existing "release" on Jenkins to make this copy from the build bucket to release bucket.

Reply:
I'm still doing some builds, so this will be the last thing that I revert.  I will make sure it's the `algorand-build` bucket.

Reply:
Actual APT .deb package serving happens from `algorand-releases` bucket, so that would be correct.

Reply:
Ok, I'll keep `algorand-releases`.

Reply:
@algobolson - we don't want to have the entire world start updating the release package just because @btoll has successfully built it. Our current staging area is `algorand-build` which is why I suggested we'll keep it.

Reply:
@tsachiherman Yes, we agree. I think the design was always to have 'build' and 'release' be separate actions, so this script could exist but be in a different Jenikns chain operated by a different button to start it? Hmm, some project maintain separate repos for 'nightly' 'beta' and 'stable'; we _could_ potentially decide to host a repo that is fully automated and not-quite-official-yet. Would "algorand-build" or another bucket be the place to host that? A publicly readable HTTP-enabled S3 bucket.